podcasts/New Search/SearchHistoryView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import PocketCastsDataModel
+import PocketCastsUtils
+
+struct SearchHistoryView: View {
+    @EnvironmentObject var theme: Theme
+
+    private var episode: Episode {
+        let episode = Episode()
+        episode.title = "Episode title"
+        episode.duration = 3600
+        return episode
+    }
+
+    init() {
+        UITableViewHeaderFooterView.appearance().backgroundView = UIView()
+    }
+
+    var body: some View {
+        VStack {
+            List {
+                HStack {
+                    Text("Recent searches")
```

Comment:
Should this be localized? or does it not matter because its a mock and this won't be the text?

Reply:
It should! Even though is a mock it's the final string, done in a75b7fe4fcfa0a19fea1c4b235bfa455d6664b3e